docs/api.md

```diff
@@ -291,24 +291,40 @@ articles.sync({
   });
 ```
 
-The synchronization updates the local data, and provides information about performed operations.
-Sample result:
+## The synchronization result object
+
+When the `#sync()` promise is fulfilled, a result object is returned, providing information about the performed operations.
+
+Here's a sample result object:
 
 ```js
 {
   ok: true,
   lastModified: 1434270764485,
-  conflicts: [], // Outgoing and incoming conflicts
-  errors:    [], // Errors encountered, if any
-  created:   [], // Created locally
-  updated:   [], // Updated locally
-  deleted:   [], // Deleted locally
-  skipped:   [], // Skipped imports
-  published: [], // Successfully published
-  resolved:  [], // Resolved conflicts, according to selected strategy
+  conflicts: [],
+  errors:    [],
+  created:   [],
+  updated:   [],
+  deleted:   [],
+  skipped:   [],
+  published: [],
+  resolved:  [],
 }
 ```
 
+The synchronization result object exposes the following properties:
+
+- `ok`: The boolean status of the synchronization operation; `true` if no unsolvable conflicts and no errors were encountered.
+- `lastModified`: The timestamp of the latest revision of server data.
+- `conflicts`: The list of encountred conflicts during both import and export operations (see *[Resolving conflicts manually](#resolving-conflicts-manually)*);
+- `errors`:    The list of encountered errors, if any.
```

Comment:
Per record (so one 401 error at the http-level can result in 20 errors at the record-level here). Is it guaranteed that a record that is listed here was left unchanged? For instance, if a record is pulled in, but the RemoteTransformer throws an error, is the lastModified value reset to (or left at) what it was, so that it will be retried next time?